server/api/appointments/getTimes.php

```diff
@@ -82,14 +121,38 @@ function calculateRemainingAppointmentsAvailable($appointmentCount, $percentAppo
 	return ceil($availableAppointmentSpots) - $appointmentCount;
 }
 
+function calculateRemainingAppointmentsAvailableForInternationalAppointment($treatyType, $numberAppointmentsAlreadyScheduled) {
+	if ($treatyType === 'china') {
+		return 15 - $numberAppointmentsAlreadyScheduled;
```

Comment:
Haha these are the values that Dr. Moody provided us as the caps for each international "station". Hard-coded because this isn't the ideal solution to this problem, but pulling them out into a config file is a good idea